typescript/tsconfig.json

```diff
@@ -0,0 +1,7 @@
+{
+  "compilerOptions": {
+    "declaration": true,
+    "baseUrl": ".",
+    "types": ["node"]
```

Comment:
Do you need this type? We generally aren't transpiling for Node.js (for the browser instead)

Reply:
We don't really have a requirement here. I just added typescript support with rules_ts and to execute a simple example.  I think, however, that line 5 is not necessary. I will remove it.